celery_rpc/utils.py

```diff
@@ -72,8 +72,6 @@ def unpack_exception(error, wrap_errors, serializer=DEFAULT_EXC_SERIALIZER):
     """
     if not wrap_errors:
         return None
-    if isinstance(error, UnpickleableExceptionWrapper):
```

Comment:
Here it is https://github.com/bourivouh/django-celery-rpc/commit/4dc2c6d7581f0c44e7aedaf66bb5c70de2ad569a#commitcomment-12645204

Or you mean comment in code?